### PR TITLE
Allow to configure additional parameters in DSL

### PR DESCRIPTION
**Pull request by mateusz-blaszkowski**

### CODE REVIEW

File: stackanetes/manifest.py [hunk 1]
```diff
@@ -37,24 +37,30 @@ def __init__(self, configuration, service_dir):
...
+        self.configmaps = configuration.get('files', [])
         self.image = configuration.get('image')
-        self.ports = configuration.get('ports', [])
         self.envs = configuration.get('envs', [])
+        self.privileged = configuration.get('privileged', True)
         self.session_affinity = configuration.get("session_affinity",[])
         self.non_root = configuration.get("non_root",[])
         self.emptydirs = configuration.get('emptyDirs', [])
         self.mounts = configuration.get('mounts', [])
-        self.envs.append({'COMMAND': self.command})
```
Comment (DTadrzak) [inline stackanetes/manifest.py]: Remove that changes. Entrypoint requires that env so it must be set. But I agree that we need to add some service_validation method.

File: stackanetes/manifest.py [hunk 2]
```diff
@@ -66,38 +72,38 @@ def _parameters_for_multi_containers_pod(self, configuration):
...
+            container_dict['configmaps'] = container_conf.get('files', [])
             config_maps.extend(container_dict['configmaps'])
-            container_dict['emptyDirs'] = container_configuration.get('emptyDirs', [])
+            container_dict['emptyDirs'] = container_conf.get('emptyDirs', [])
             empty_dirs.extend(container_dict['emptyDirs'])
-            container_dict['mounts'] = container_configuration.get('mounts', [])
+            container_dict['mounts'] = container_conf.get('mounts', [])
             mounts.extend(container_dict['mounts'])
-            container_dict['envs'].append(
-                {'COMMAND': container_configuration['command']})
```
Comment (DTadrzak) [inline stackanetes/manifest.py]: ditto

**Comment by DTadrzak** [comment]
lgtm only that one nit. 
Branch tested.
What do you @PiotrProkop  @ss7pro  think?

File: templates/generic-daemonset.yml.j2 [hunk 3]
```diff
@@ -17,8 +44,10 @@ spec:
         {{ label }}: enabled
       securityContext:
         runAsUser: 0
-      hostNetwork: true
       dnsPolicy: ClusterFirst
+{%- if host_network %}
+      hostNetwork: true
```
Comment (PiotrProkop) [inline templates/generic-daemonset.yml.j2]: Please add this flag to all currently deployed daemonsets with host network.

File: templates/generic-daemonset.yml.j2 [hunk 4]
```diff
@@ -38,10 +67,12 @@ spec:
             path: {{ mount['host_path'] }}
 {%- endfor %}
       containers:
-{% for container in containers %}
+{%- for container in containers %}
         - name: {{ container['name'] }}
+{%- if container['privileged'] %}
```
Comment (PiotrProkop) [inline templates/generic-daemonset.yml.j2]: Please add this flag to all currently prepared daemonsets with privileged containers.

**Comment by PiotrProkop** [comment]
LGTM

**Pull request merged**